.github/workflows/asciidoc-build.yml

```diff
@@ -0,0 +1,72 @@
+name: AsciiDoc Documentation Build
+
+on:
+  push:
+
+
+jobs: 
+  asciidoctor-build:
+    name: Run Asciidoctor on Documentation
+    
+    runs-on: ubuntu-18.04
+
+    steps:
+    # Checkout osi documentation
+    - name: Checkout osi-documentation
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    # Checkout all other OSI repositories with branch "migration" and nested into osi-documentation
+    - name: Checkout open-simulation-interface
+      uses: actions/checkout@v2
+      with:
+        repository: OpenSimulationInterface/open-simulation-interface
+        ref: migration
+        path: open-simulation-interface
+
+    - name: Checkout OSI Sensor Model Packaging
+      uses: actions/checkout@v2
+      with:
+        repository: OpenSimulationInterface/osi-sensor-model-packaging
+        ref: migration
+        path: osi-sensor-model-packaging
+
+
+    - name: Checkout OSI Validation
+      uses: actions/checkout@v2
+      with:
+        repository: OpenSimulationInterface/osi-validation
+        ref: migration
+        path: osi-validation
+
+
+    - name: Checkout OSI Visualizer
+      uses: actions/checkout@v2
+      with:
+        repository: OpenSimulationInterface/osi-visualizer
+        ref: migration
+        path: osi-visualizer
+
+
+    - name: Checkout proto2cpp
+      uses: actions/checkout@v2
+      with:
+        repository: OpenSimulationInterface/proto2cpp
+        ref: migration
+        path: proto2cpp
+
+       
+    # run Asciidoctor
+    - name: Build HTML
+      uses: avattathil/asciidoctor-action@master
+      with:
+        program: "asciidoctor -D . --failure-level WARN -r asciidoctor-diagram -a mathjax --trace --backend=html5 index.adoc -o documentation.html"
```

Comment:
I would recommend a more descriptive name for the output file, for example, "open-simulation-interface.html".

Reply:
Well, yes you are correct.
But in contrast to the other ASAM standards, the filename is not really important. It is just deployed to gh-pages and never (?) downloaded. To be tidy I'm going to change that and also look at overall versioning when at it.

Reply:
change the filename to open-simulation-interface_[version from git].html with the version from 'git describe'
E.g. "documentation_v1.0.0-18-gb150a36.html". This can be read as: 18 commits ahead tag v1.0.0. "b150a36" is the commit hash of this build.
This gives both a definitive versioning using the commit hash and a human-readable version with number of commits ahead of last tag.

Reply:
Done. Versioning is used in filename and also inserted into asciidoc.